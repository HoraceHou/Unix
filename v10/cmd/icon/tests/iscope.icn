procedure main()
   write(Dword(&null))
   write(Vword(123))
   write(Symbol("globals"))
   (1 to 3) & (1 to 3) & write(Ivar("ilevel"))
   write(Ivar("sp"))
   write(Ivar("gfp"))
   foo()
end

procedure foo()
   write(Ivar("pfp"))
   write(Ivar("argp"))
   write(Ivar("fussbudget"))
end
