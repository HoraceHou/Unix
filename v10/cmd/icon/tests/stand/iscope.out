-2147483648
123
82316
3
82908
0
82896
82888
Run-time error 205 at line 14 in iscope.icn
value out of range
offending value: "fussbudget"
